### ISSUE TITLE
Variable bounds not recorded with SqliteRecorder

### ISSUE THREAD

**Issue by imcovangent** [OpenMDAO/OpenMDAO#693]
Taking the latest version of OpenMDAO at the moment I am not able to collect the lower and upper bounds of variables from the recorder. As a test I use the example in the documentation (found [here](http://openmdao.org/twodocs/versions/2.3.0/features/recording/getting_metadata.html)). Hence I use the following test script:
```
from openmdao.api import Problem, ScipyOptimizeDriver, SqliteRecorder, CaseReader
from openmdao.test_suite.components.sellar import SellarDerivatives

import numpy as np

prob = Problem(SellarDerivatives())

prob.model.add_design_var('z', lower=np.array([-10.0, 0.0]),
                               upper=np.array([10.0, 10.0]))
prob.model.add_design_var('x', lower=0.0, upper=10.0)
prob.model.add_objective('obj')
prob.model.add_constraint('con1', upper=0.0)
prob.model.add_constraint('con2', upper=0.0)

prob.driver = ScipyOptimizeDriver()

prob.driver.recording_options['record_metadata'] = True

recorder = SqliteRecorder("cases.sql")
prob.driver.add_recorder(recorder)

prob.setup()
prob.run_driver()

prob.cleanup()

cr = CaseReader("cases.sql")

z_lower = cr.output2meta['z']['lower']
z_upper = cr.output2meta['z']['upper']
```

For `z_lower` and `z_upper` I expected to retrieve the arrays that were specified with `prob.model.add_design_var('z', lower=np.array([-10.0, 0.0]), upper=np.array([10.0, 10.0]))`, but instead I simply get `None` returned. Is this an issue or am I missing something in how I should retrieve the specified bounds from the recorder?

**Comment by kmarsteller** [comment]
Your issue has been noted, and will likely be fixed in OpenMDAO v2.4, coming soon. We shall keep you posted.

**Comment by imcovangent** [comment]
Thanks! :)

**Comment by kmarsteller** [comment]
It didn't make it into 2.4, but someone is working this issue now. So I hope to have good news for you on this issue soon. Sorry for the inconvenience.

**Comment by imcovangent** [comment]
No problem, thanks for letting me know.

**Comment by JustinSGray** [comment]
fixed by #724
